app/Http/Controllers/PlansController.php

```diff
@@ -21,8 +21,7 @@ public function __construct()
     public function index(Request $request)
     {
         $plans = Plan::excludeArchive()->search('"'.$request->input('search').'"')->paginate(10);
-        $planTotal = Plan::excludeArchive()->search('"'.$request->input('search').'"')->get();
-        $count = $planTotal->count();
+        $count = Plan::excludeArchive()->search('"'.$request->input('search').'"')->count();
```

Comment:
well actually you should be able to use `$plans->totals()` which is the same query but only with pagination

`$plans` and `$count` are two identical queries